compiler/rustc_builtin_macros/messages.ftl

```diff
@@ -197,6 +197,8 @@ builtin_macros_format_redundant_args = redundant {$n ->
 
 builtin_macros_format_remove_raw_ident = remove the `r#`
 
+builtin_macros_format_reorder_format_parameter = did you mean `{$replacement}`?
```

Comment:
For future reference, we prefer not to ask questions of the user anymore. If I were writing this message, I'd say something like "you might have meant to write `#?`".